src/components/inputs/Checkbox/CheckboxSet.tsx

```diff
@@ -34,7 +35,8 @@ function CheckboxSetComponent({
 
   const [checks, setChecks] = useState(all(false));
 
-  if (!validate(children, 'checkbox')) return null;
+  if (!validate(children, 'checkbox'))
+    console.warn('Unable to valid children on CheckboxSet');
```

Comment:
1. Did you mean "validate" rather than "valid"?
2. Please only enable this warn in dev environments.

Reply:
Thanks, updated! Also fixed `onKeyUp` prop for keyboard events

Reply:
Allowing `onKeyUp` callbacks does not fix the issues the keyboard controls. Let's remove those changes and address the keyboard issues in a separate PR.

Reply:
Here's an example:

### control via pointer events

https://user-images.githubusercontent.com/319302/160462160-aca4fd9b-b57d-4298-b013-fddcf0455d0b.mov

### control via keyboard events

https://user-images.githubusercontent.com/319302/160462217-5e8fbf2d-2e9b-4db5-b164-e5b879047118.mov

Reply:
Ahh thanks I see, the children checkboxes aren't updating with keyboard control. I'll submit it a new issue for this bug. Reverted the `onKeyUp` changes in this PR.